recipes/kivy/meta.yaml

```diff
@@ -0,0 +1,109 @@
+{% set name = "kivy" %}
+{% set version = "1.10.1.dev0" %}
+{% set sha256 = "8b29033a9e5700120ad2bf88529356fe48783a7fed6a88110e20a033790694ec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/kivy/kivy.git
```

Comment:
Is there a tag we can use?

Reply:
Oh, right, I forgot. I'll try the last release. So kivy tends to do releases once a year (or more), but there's always a lot of work going on in master. I was planning to keep forge updated with the last release, even if it's old. 

I was wondering though, is there a way to do e.g (semi-)nightly releases somewhere. Would I just create a kivy channel for it or is there a way do also do this in forge? I suppose we could also just tell users to first install the last release from conda and then do a pip upgrade from master.

Reply:
Turns out that the last kivy release is not supported because the version of cython it requires is not compatible with python 3.6 according to conda pinning. So I can either wait for the next release and then finish this PR, or do an initial release with master just to get it into conda and then after the next kivy release fix the recipe to only use a release rather than master.